specs/DesignDiscussions/InputLabelsOptionLabel.md

```diff
@@ -0,0 +1,440 @@
+	"label": 
+	{
+		"type": "TextBlock",
+		"text": "Please enter your name:",
+		"weight": "Bolder",
+		"color": "Good"
+	},
+	"placeholder": "Last, First"
+}
+```
+
+The problem with this approach, however, is that if we allow the host to specify formatting, there could be a conflict between the formatting provided by the card author and that provided by the host. For example, if the card author makes all their labels bold, the host's indication that required labels should be bold will not provide any additional information, and the end user will not see any indication of which fields are required.
+
+##### Non-adjacent labels
+
+Following this approach, non-adjacent labels are not going to be possible as labels are going to be tied to the inputs which will make it easier to users to know what label corresponds to each input.
+
+As this approach sets the rules of how labels are rendered, it also limits where a label can be drawn in valid scenarios as the next ones:
+
+![img](assets/InputLabels/LabelsToTheLeft.png)
+
+I built this card using a series of column sets. Although it is an acceptable UI, this is probably not the approach we want for card authors to label their inputs on the side. The card authoring is fairly non-intuitive to get a good result. If we want to support labels to the left of the input, we should probably do this as a first-class explicit property (```"labelPosition"``` or ```"allowHorizontalLabels"```). 
+
+This card is another possible example of a use case for non-adjacent labels, where there is additional information between the label and the input:
+
+![img](assets/InputLabels/AdditionalLabelContent.png)
+
+This scenario could be covered by a RichTextBlock that contains both pieces of text being used as the label (arguably the entire text should be read as the accessibility label for the input). 
+
+This approach will limit what elements can be rendered as the label to only text.
+
+## Backwards Compatibility
+On earlier versions of renderers the label would be completely dropped, leading to a bad user experience. The card may be completely unusable if without the labels the user cannot deduce what information is meant to be provided in the unlabeled inputs.
+
+Because the label is a property, not a new type, card authors would not be able to use type to trigger fallback to an independent text block and input. To support this, the card author would need to use requires as follows:
+
+```json
+{
+	"type": "Input.Text",
+	"id": "NameId",
+	"label": "Please enter your name",
+	"requires": {
+		"adaptiveCards": "1.3"
+	},
+	"fallback": {
+		"type": "Container",
+		"items": [
+			{
+				"type": "TextBlock",
+				"text": "Please enter your name:"
+			},
+			{
+				"type": "Input.Text",
+				"id": "NameId"
+			}
+		]
+    }
+}
+```
+We could also possibly add a specific named feature to the requires dictionary to allow card authors to address it directly using something like:
+```json
+"requires": {
+	"inputLabels": "1.0"
```

Comment:
I quite like this latter very specific approach. it can make things simpler to use for the card author/improves readability of the card json and we dont have to remember the AC schema version to feature mapping necessarily. @paulcam206 / @matthidinger / @dclaux  thoughts?